src/components/BankTransactions/BankTransactions.tsx

```diff
@@ -72,6 +74,7 @@ export interface BankTransactionsProps {
   filters?: BankTransactionFilters
   hideHeader?: boolean
   stringOverrides?: BankTransactionsStringOverrides
+  convertMatchDetailsToInAppLink?: (details: MatchDetailsType) => ReactNode
```

Comment:
I have a strong, strong desire not to expose the `MatchDetailsType` (or analogous types) here. I would _love_ to settle on a standard set of fields that are provided  (I'd suggest entity type, id, external id, reference number, `metadata`) and then always provide that. And if we don't have that in these types, add it in on our side so we can have that standard experience everywhere.